projects/2019_12_14_pkt-whitepaper-website.md

```diff
@@ -372,3 +372,589 @@ The applicant understands that the Network Steward is not a legal entity and no
+PKT intends to change the dynamic from pay-for-access to pay-for-priority because it makes absolutely no sense that a person should be unable to communicate while there's a perfectly good Internet connection sitting idle.
+This plan is obviously very ambitious, but like ARPANET started as a couple of computers connected over the telephone PKT has begun this way as well.
+PKT also carries forward the belief that all communications should be encrypted, no matter what. This means, nobody should have to apply for https certificates, there should be no question of the origins of a data packet when it is received, and nobody should ever need a VPN to get another Internet connection when they already have one.
+If you believe that the internet should be owned by the people, and that no one should ever be denied access while bandwidth sits unused, if you want to look back on the days of $50 a month the way we look back on the days of $18 phone calls, come build with us. We're just getting started.
+Button: PKT Roadmap
+Button: PKT Origin Story
+
+#PKT Roadmap 
+There are four distinct stages of evolution which the PKT project will undergo. These stages may occur out of order at times, but the project will be viable in every state, just as a caterpillar is a viable creature before it becomes a butterfly.
+
+##The PKT Roadmap is:
+Build a VPN market where anyone can participate and the minimum price is free
+Support low cost Internet sharing hardware that allows people to earn PKT using their extra bandwidth
+Create a bandwidth market where there are virtual ISPs to do the network engineering so that building a mesh network is easy
+Support high performance networking hardware which automatically sells its bandwidth into the decentralized bandwidth market
+
+##Stage One, VPN Marketplace
+  
+Price will be decided by auction with a minimum bid of zero. In order to stay decentralized and achieve low enough cost of transactions to enable true micropayments, VPNs will be purchased using PKT, a cryptocurrency which is mined by performing encryption operations and expending Internet bandwidth. Anyone who wants to use a VPN, create their own, or mine PKT will easily be able to do so.
+
+##Stage Two, Support Internet Sharing
+  
+Once there are approximately one million installations of the VPN client application, PKT will begin supporting third party operators who offer internet sharing technology and hardware for home or business users. This technology and hardware will allow people to earn PKT with their unused internet bandwidth and like the VPN market, it will have a minimum bid of zero. Since it will be the VPN client application that connects to the Internet sharing hardware, there will be no privacy risk nor legal risk because all data will be sent through the VPN.
```

Comment:
>nor legal risk

May need to consult a lawyer before making this claim. Uncertain about global jurisdictions.

Reply:
Perhaps:

> , all data will go through the VPN, solving many privacy and legal concerns with shared internet access.